apps/menu-bar/electron/src/main.ts

```diff
@@ -51,6 +51,11 @@ const createMainWindow = () => {
     mainWindow.loadURL(`file://${path.join(__dirname, '../../../app/dist/index.html')}`);
   }
 
+  mainWindow.webContents.once('dom-ready', () => {
+    // Only keep the current session in the logs
+    mainWindow.webContents.executeJavaScript("localStorage.setItem('logs', '[]')");
+  });
```

Comment:
We need to reset localStorage logs on launch to ignore the logs from the previous session